src/common/snippets/tests/src/pass/gn_tokenization.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include <pass/gn_tokenization.hpp>
+#include "snippets/pass/gn_tokenization.hpp"
+#include "common_test_utils/common_utils.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+std::string TokenizeGNSnippetsTests::getTestCaseName(testing::TestParamInfo<GroupNormalizationParams> obj) {
+    PartialShape input_shape;
+    size_t num_group;
+    float eps;
+    std::tie(input_shape, num_group, eps) = obj.param;
+    std::ostringstream result;
+    result << "IS=" << ov::test::utils::partialShape2str({input_shape}) << "_";
+    result << "num_group=" << num_group << "_";
+    result << "eps=" << eps;
+    return result.str();
+}
+
+void TokenizeGNSnippetsTests::SetUp() {
+    TransformationTestsF::SetUp();
+    PartialShape data_shape;
+    size_t num_group;
+    float eps;
+    std::tie(data_shape, num_group, eps) = this->GetParam();
+    OPENVINO_ASSERT(data_shape.size() >= 2, "First input rank for group normalization op should be greater than 1");
+    PartialShape scaleShiftShape = PartialShape{data_shape[1]};
+    std::vector<PartialShape> input_shapes = { data_shape, scaleShiftShape, scaleShiftShape};
+    snippets_model = std::make_shared<GroupNormalizationFunction>(input_shapes, num_group, eps);
+    manager.register_pass<ov::snippets::pass::TokenizeGNSnippets>();
```

Comment:
Theoretically the `SnippetsMarkSkipped` could also affect GN tokenization by accidentally marking the node as skipped, for example. Moreover
Do you think it's worth to add the mark_skipped pass here as well?

Also, I don't see a check that the node is not skipped (like [this](https://github.com/openvinotoolkit/openvino/blob/68bdbbe0fba08386677071c83018be9d07f58916/src/common/snippets/src/pass/collapse_subgraph.cpp#L258)) in the GN tokenization pass. Please add it, since we should not tokenize any nodes marked as skipped.

Reply:
@IvanNovoselov ,I add the skip check in the gn tokenization[ccaf81b](https://github.com/openvinotoolkit/openvino/pull/23903/commits/ccaf81b6f7db904266b20acf39a39afe9e4caea1). But for test for now there is no SnippetsMarkSkipped involved in GN. We can add test if it exist in the future, maybe a different test for specific backend, not in common tokenization, as they have different SnippetsMarkSkipped check. What do you think? Thank you Ivan!